astro/src/content/docs/lifecycle/register-users/advanced-registration-forms.mdx

```diff
@@ -175,23 +176,23 @@ Create a second step. Add your custom house hunting parameter fields:
 
 After you've added these fields to the form, feel free to rearrange the form fields within each step by clicking the arrows to move a field up or down. 
 
-The form configuration specifies steps and field display order within those steps. If you need to move a field between steps, delete it from one step and add it to another. To change field validation, return to the <strong>Fields</strong> section and make your changes. When you're done tweaking the form to your liking, save it.
+The form configuration specifies steps and field display order within those steps. If you need to move a field between steps, delete it from one step and add it to another. To change field validation, return to the **Fields** section and make your changes. When you're done tweaking the form to your liking, save it.
```

Comment:
I think `Fields` should use `Breadcrumb` as well. The section header is related to navigation.

Reply:
Fixed